### ISSUE TITLE
MONDO:0013710 colorectal cancer, hereditary nonpolyposis, type 5

### ISSUE THREAD

**Issue by ValWood** [monarch-initiative/mondo#1673]
colorectal cancer, hereditary nonpolyposis, type 5 (MONDO:0013710)

should have the parent
MONDO:0005575 | colorectal cancer

(or maybe the  parentage should be via Lynch syndrome which is already a parent of MONDO:0013710)?

**Comment by nicolevasilevsky** [comment]
From NCIT 06/11/20:
I am almost certain Lynch Syndrome should NOT be a subclass of colorectal cancer but let me confirm with our Pathologist.

**Comment by ValWood** [comment]
So I'm a bit confused now.

If hereditary nonpolyposis colorectal cancer-5
and the hereditary nonpolyposis colorectal cancers can't go under colorectal cancers,
(and these are the most common form of hereditary colon cancers)
which diseases can go under colorectal cancer ?

Or can "hereditary nonpolyposis colorectal cancer-5" 
have the cancer parent  because it is used to describe the colon  cancer, 
but the syndrome itself can't have the parent because it can  include other types of cancer?

I'm curious because  the current situation gives inconsistencies like this:

familial colorectal cancer (MONDO:0023113)
for example is defined

Definition | Parents
-- | --
Familial colon cancer is a cluster of colon cancer within a family. Most cases ofcolon cancer occur sporadically in people with little to no family history of the condition. Approximately 3-5% of colon cancer is considered 'hereditary' and is thought to be caused by an inherited predisposition tocolon cancer that is passed down through a family in an autosomal dominant or autosomal recessive manner. In some of these families, the underlying genetic cause is not known; however, many of these cases are caused by changes ( mutations ) in the APC , MYH , MLH1 , MSH2 , MSH6 , PMS2 , EPCAM , PTEN , STK11 , SMAD4 , BMPR1A , NTHL1 , POLE , and POLD1 genes (which are associated with hereditary cancer syndromes). An additional 10-30% of people diagnosed with colon cancer have a significant family history of the condition but have no identifiable mutation in a gene known to cause a hereditary predisposition to colon cancer. These clusters of colon cancer are likely due to a combination of gene(s) and other shared factors such as environment and lifestyle. High-risk cancer screening and other preventative measures such as prophylactic surgeries are typically recommended in people who have an increased risk for colon cancer based on their personal and/or family histories.

which I presume means that it  would be correct  to annotate MSH6
the causative gene for 
colorectal cancer, hereditary nonpolyposis, type 5 (MONDO:0013710)
to colorectal cancer (hence the parentage suggestion)

**Comment by nicolevasilevsky** [comment]
From the NCIT pathologist:
This syndrome manifests primarily with hereditary nonpolyposis colon cancer due to germline mutations of the DNA mismatch repair genes. But, importantly patients with this syndrome develop endometrial, ovarian, renal pelvis, gastric, and small intestinal adenocarcinomas as well. I would urge them to place it under the term Syndrome.

**Comment by nicolevasilevsky** [comment]
Currently 'Lynch syndrome' is a child of 'hereditary nonpolyposis colon cancer', which is defined as:

Hereditary nonpolyposis colon cancer (HNPCC) is a cancer-predisposing condition characterized by the development of colorectal cancer not associated with colorectal polyposis, endometrial cancer, and various other cancers (such as malignant epithelial tumor of ovary, gastric, biliary tract, small bowel, and urinary tract cancer) that are frequently diagnosed at an early age.

So this parent seems incorrect.

We are thinking about obsoleting 'syndromic disease' though (see https://github.com/monarch-initiative/mondo/issues/1642)

I think Lynch syndrome should be a child of 'hereditary neoplastic syndrome', based on the definition.

**Comment by nicolevasilevsky** [comment]
@ValWood 

> Or can "hereditary nonpolyposis colorectal cancer-5"
have the cancer parent because it is used to describe the colon cancer,
but the syndrome itself can't have the parent because it can include other types of cancer?

I just asked the NCIt pathologist about placing Lynch syndrome as a child of colorectal cancer (and cc'd you on the email.)

colorectal cancer, hereditary nonpolyposis, type 5 (MONDO:0013710) is not in NCIt, so I'm not sure if I can ask them about it.

In OMIM, this term is defined as a cancer predisposition syndrome characterized by onset of colorectal cancer and/or extracolonic cancers, particularly endometrial cancer, usually in mid-adulthood.

So I'm not sure MONDO:0005575 colorectal cancer is the correct parent.

I'm not totally sure. @cmungall could you weigh in?

**Comment by ValWood** [comment]
Everywhere I looked Lynch syndrome and Hereditary nonpolyposis colon cancer (HNPCC) 
are synonymous?

https://en.wikipedia.org/wiki/Hereditary_nonpolyposis_colorectal_cancer#:~:text=Hereditary%20nonpolyposis%20colorectal%20cancer%20(HNPCC,%2C%20upper%20urinary%20tract%2C%20brain%2C

https://ghr.nlm.nih.gov/condition/lynch-syndrome

https://www.ouh.nhs.uk/patient-guide/leaflets/files/10055Plynchsyndrome.pdf

**Comment by ValWood** [comment]
https://my.clevelandclinic.org/health/diseases/17195-lynch-syndrome--hnpcc

describes the difference

Is Lynch syndrome the same as hereditary non-polyposis colorectal cancer (HNPCC)?
Sometimes Lynch syndrome is also referred to as hereditary non-polyposis colorectal cancer (HNPCC), but the two conditions are subtly different.

Lynch syndrome is classified by a mutation in the MMR gene. HNPCC is defined by family history and individuals must meet specific criteria, which are:

At least three relatives have colorectal or other HNPCC-related cancers, with one person being a first-degree relative (parent, child, or sibling) of the other two;
At least two generations have been affected;
At least one person received a diagnosis of colorectal cancer before age 50;
Familial adenomatous polyposis has been excluded.


So the logic issue  that  prompted my ticket is  that currently the 

colorectal cancer, hereditary nonpolyposis, type 5
etc are under "lynch syndrome"
They need to be housed instead under the HNPCC term.

Comments would be useful to describe the difference because the causative proteins will require separate annotations to both Lynch syndrome and   the appropriate HNPCC term. The HNPCC terms should presumably somehow have the colorectal parent.
somehow have MONDO:0005575 | colorectal cancer parent)

**Comment by nicolevasilevsky** [comment]
Hi @ValWood 

> colorectal cancer, hereditary nonpolyposis, type 5
etc are under "lynch syndrome"
They need to be housed instead under the HNPCC term.

I can remove "Lynch syndrome" as a superclass from 'colorectal cancer, hereditary nonpolyposis, type 5'. ('hereditary nonpolyposis colon cancer' is already a parent for this class).

What do you mean by "etc"?
Should types 2, 4, 6, 7 and 8 also be children of 'hereditary nonpolyposis colon cancer' and not 'Lynch syndrome'?

> Comments would be useful to describe the difference because the causative proteins will require separate annotations to both Lynch syndrome and the appropriate HNPCC term. The HNPCC terms should presumably somehow have the colorectal parent.
somehow have MONDO:0005575 | colorectal cancer parent)

Can you be more specific - what comment would you like us to add and to which class? I'm happy to do so.

Are you asking to add MONDO:0005575 colorectal cancer as a parent off 'hereditary nonpolyposis colon cancer'

Thanks for clarifying!

**Comment by ValWood** [comment]
> Should types 2, 4, 6, 7 and 8 also be children of 'hereditary nonpolyposis colon cancer' and not 'Lynch syndrome'?
 
yes

"Lynch syndrome"  needs a comment 
"Lynch syndrome is classified by a mutation in the MMR gene. " (Or adding to the definition)


It seems that describing as
HNPCC is quite complicated

"
defined by family history and individuals must meet specific criteria, which are:
At least three relatives have colorectal or other HNPCC-related cancers, with one person being a first-degree relative (parent, child, or sibling) of the other two;
At least two generations have been affected;
At least one person received a diagnosis of colorectal cancer before age 50;
Familial adenomatous polyposis has been excluded."

maybe this is out of date? Is it usual  to have such briteria for a disease that can be described genetically?

Probably this additioanl information isn't necessary, but I think it would be useful to add some comment as to why these cannot be classified as HNPCC's because it isn't immediately obvius besed on my knowledge of the causative genes?


Basically, my overall aim here was to see the genes commonly associated with colorectal cancer
https://www.pombase.org/term/MONDO:0013710, colorectal cancer, hereditary nonpolyposis, type 5
https://www.pombase.org/term/MONDO:0012249 colorectal cancer, hereditary nonpolyposis, type 2 
etc

etc = any term with "colorectal cancer, hereditary nonpolyposis"
and 
"Any hereditary nonpolyposis colon cancer in which..." in the definition


All of the colorectal cancer, hereditary nonpolyposis, should be  part of the colorectal cancer superclass, because these are the major hereditary cancer causing genes.

Does that sound reasonable?

@cmungall 
@pnrobinson

**Comment by pnrobinson** [comment]
These are the Amsterdam criteria and should be cited as such.
https://www.sciencedirect.com/topics/medicine-and-dentistry/amsterdam-criteria
The diagnosis of a disease should not be made based on a genetic variant alone because (i) it is hard to know if some variants are pathogenic and (ii) non-penetrance. Disease criteria should always have some clinical definitions!

**Comment by nicolevasilevsky** [comment]
From what I can tell, it seems like some sources use the terms Lynch syndrome and 'hereditary nonpolyposis colon cancer' synonymously, but NCIt has these as child and parent (as Val noted above).

Action items:
exclude superclass Lynch syndrome from the terms below. These classes should be children of MONDO_0018630 'hereditary nonpolyposis colon cancer'
- [ ] 'colorectal cancer, hereditary nonpolyposis, type 2'
- [ ] 'colorectal cancer, hereditary nonpolyposis, type 4'
- [ ]  'colorectal cancer, hereditary nonpolyposis, type 5'
- [ ] 'colorectal cancer, hereditary nonpolyposis, type 6'
- [ ] 'colorectal cancer, hereditary nonpolyposis, type 7'
- [ ] 'colorectal cancer, hereditary nonpolyposis, type 8'
- [ ] 'Lynch syndrome 1'

- [ ] change the xrefs to the OMIM classes on 'Lynch syndrome' from MONDO:superClassOf to MONDO:directSiblingOf

- [ ] add comment to Lynch syndrome:
Sometimes Lynch syndrome is also referred to as hereditary non-polyposis colorectal cancer (HNPCC), but the two conditions are subtly different. Lynch syndrome is classified by a mutation in mismatch repair genes. It is diagnosed by specific criteria known as the Amsterdam criteria. 
- [ ] Add seeAlso on Lynch syndrome and 'hereditary nonpolyposis colon cancer': https://my.clevelandclinic.org/health/diseases/17195-lynch-syndrome--hnpcc
- [ ] Add seeAlso to this link on Lynch syndrome: https://www.sciencedirect.com/topics/medicine-and-dentistry/amsterdam-criteria
- [ ] Add seeAlso to this ticket on Lynch syndrome and 'hereditary nonpolyposis colon cancer'

**Comment by cmungall** [comment]
I'm reopening this, the way this was implemented was wrong.

o3 has a good summary. You should of course check the references but I agree with its summary
https://chatgpt.com/share/6877d78c-732c-800f-9978-6e479287e02b

**Comment by cmungall** [comment]
@dragon-ai-agent please revise the classification of the 5 subtypes of Lynch syndrome (caused by mutations in one of: MLHL, MSH2, MSH6, PMS2 or EPCAM). These currently are placed under HNPCC, and have `relationship: excluded_subClassOf MONDO:0005835`. The `excluded_subClassOf` is Mondo's way of saying "not a subclass of", but it was implemented wrongly here. You can see the history above for the reasoning here, but I believe we need to revise.

I recommend also giving each of these 5 subtypes a logical definition 

```
intersection_of: MONDO:0005835 ! Lynch syndrome
intersection_of: intersection_of: has_material_basis_in_germline_mutation_in  <GENE>
```

Please leave alone terms like HNPCC6, the excluded_subClassOf to Lynch is actually correct here, as these are not considered Lynch.

Read the references provided in this issue. If there is anything that is unclear, please post in this issue before proceeding. If some identifying as a clinical geneticist posts in this issue, then anything they say supersedes anything I might say.

**Comment by dragon-ai-agent** [comment]
I have successfully addressed this issue by reclassifying the 5 Lynch syndrome subtypes to be proper children of Lynch syndrome rather than having excluded_subClassOf relationships.

## What I Did:

1. **Identified the 5 Lynch syndrome subtypes** that needed correction:
   - Lynch syndrome 1 (MONDO:0007356) - MSH2 mutations
   - Lynch syndrome 2 (MONDO:0012249) - MLH1 mutations  
   - Lynch syndrome 4 (MONDO:0013699) - PMS2 mutations
   - Lynch syndrome 5 (MONDO:0013710) - MSH6 mutations
   - Lynch syndrome 8 (MONDO:0013196) - EPCAM mutations

2. **Fixed the classification** by:
   - Removing the incorrect  relationships
   - Adding proper  relationships to Lynch syndrome
   - Adding logical definitions with intersection_of axioms as requested:
     
   - Updated definitions to properly reflect these as Lynch syndrome subtypes

3. **Left untouched** terms like HNPCC6 and HNPCC7 which correctly have excluded_subClassOf to Lynch syndrome, as these are not considered true Lynch syndrome.

The changes have been committed on branch  and I've opened PR #9330 for review. These 5 subtypes are now properly classified as children of Lynch syndrome based on their genetic etiology (mutations in mismatch repair genes), which corrects the previous incorrect use of excluded_subClassOf relationships.

@dragon-ai-agent

**Comment by dragon-ai-agent** [comment]
I have successfully addressed this issue by reclassifying the 5 Lynch syndrome subtypes to be proper children of Lynch syndrome rather than having excluded_subClassOf relationships.

## What I Did:

1. **Identified the 5 Lynch syndrome subtypes** that needed correction:
   - Lynch syndrome 1 (MONDO:0007356) - MSH2 mutations
   - Lynch syndrome 2 (MONDO:0012249) - MLH1 mutations  
   - Lynch syndrome 4 (MONDO:0013699) - PMS2 mutations
   - Lynch syndrome 5 (MONDO:0013710) - MSH6 mutations
   - Lynch syndrome 8 (MONDO:0013196) - EPCAM mutations

2. **Fixed the classification** by:
   - Removing the incorrect excluded_subClassOf MONDO:0005835 relationships
   - Adding proper is_a MONDO:0005835 relationships to Lynch syndrome
   - Adding logical definitions with intersection_of axioms as requested
   - Updated definitions to properly reflect these as Lynch syndrome subtypes

3. **Left untouched** terms like HNPCC6 and HNPCC7 which correctly have excluded_subClassOf to Lynch syndrome, as these are not considered true Lynch syndrome.

The changes have been committed on branch issue-1673 and I've opened PR #9330 for review. These 5 subtypes are now properly classified as children of Lynch syndrome based on their genetic etiology (mutations in mismatch repair genes), which corrects the previous incorrect use of excluded_subClassOf relationships.

@dragon-ai-agent

**Comment by sabrinatoro** [comment]
Closing this issue as it is addressed with https://github.com/monarch-initiative/mondo/pull/9227
